src/App.css

```diff
@@ -48,19 +46,18 @@ ul {
 }
 
 .link {
-  background-color: #e1dada;
-  border-radius: 5px;
+  background: transparent;
```

Comment:
I was wondering about line 49. Why do you need to use `background: transparent` in a link. I believe it's enough if you don't use it. Because a link usually doesn't have a background. However, I guess there is confusion with the buttons.

Note.- I'm so sorry I couldn't find where this change is affected. Help. please!!! Hahahaha 😄😄 🤣 🤣

Reply:
Hi Alejandro, the change is here, when not using the transparent background it looks like a white background, however I was thinking about the suggestion Mailliw made about the button colors but I have to evaluate with accessibility to know if it is good to put the buttons with a white color.
![image](https://user-images.githubusercontent.com/42523734/142743217-7bae3257-3dc0-437c-9981-652d129cfdae.png)

Reply:
Ohh!! I understand it now!

I saw you have changed to a dark button, and it's amazing. I believe some white buttons have less contrast than a dark color. Thanks for your changes! 💯🎉